cassandra/src/main/java/org/apache/calcite/adapter/cassandra/CassandraEnumerator.java

```diff
@@ -83,8 +83,10 @@ private Object currentRowField(int index, SqlTypeName typeName) {
       return current.getInt(index);
     } else if (type == DataType.bigint()) {
       return current.getLong(index);
-    } else if (type == DataType.cdouble() || type == DataType.cfloat()) {
+    } else if (type == DataType.cdouble()) {
       return current.getDouble(index);
+    } else if (type == DataType.cfloat()) {
+      return current.getFloat(index);
```

Comment:
I am not sure why github is comparing my commit with older version of master branch. It seems my branch was compared to master at commit [[CALCITE-1848] Rename MySource to FileSource (Darion Yaphet)](https://github.com/apache/calcite/commit/d5c713c5cd447e261e3ae5cffa728c073f425e28)I.  compared my change with current master branch and mine is the same as master. I guess combining merge and change commit into one commit trigger some bugs in github diff.